Engine/source/console/consoleObject.h

```diff
@@ -542,6 +545,19 @@ class AbstractClassRep : public ConsoleBaseType
 
 extern AbstractClassRep::FieldList sg_tempFieldList;
 
+inline AbstractClassRep* AbstractClassRep::getClassRepFromName(const char* className)
+{
+	AbstractClassRep* current = classLinkList;
+	while(current != NULL)
+	{
+		if(dStricmp(current->getClassName(),className) == 0)
```

Comment:
I'd bet my money on the string comparison since there are tons of classes with varying name lengths in Torque. ;)